### PR TITLE
PLAT-61631 - Added Docs for Interoperability

### PR DESCRIPTION
**Pull request by rundmt**
### Issue Resolved / Feature Added
Added Docs for Interoperability with third-party libraries

### Links
PLAT-61631

Enact-DCO-1.0-Signed-off-by: Derek Tor <derek.tor@lge.com>

### CODE REVIEW

File: docs/interoperability.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## Overview
+
+`Enact` is a fantastic way to build apps. By default we give you a template using our `Moonstone`
+library. `Moonstone` is a library designed for TVs. However, if you're not looking to create TV
+apps, you can easily use `Enact` to build applications for desktop and mobile. You can mix and match
+third-party libraries to use with `Enact` easily. Alternatively, it is also straightforward if you
+just want to use `Enact` components in your own `React` app.
+
+## Using Third-Party Components inside of Enact
+When creating an app using `enact create`, it creates an `Enact` boilerplate that setups for a
```
Comment (webOS101) [inline docs/interoperability.md]: reword.

File: docs/interoperability.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+`Moonstone` styled app. However if you want to use something outside of `Moonstone` like
+`material-ui`, `reactstrap`, or `react-router` just use `npm install`.
+
+You can include components just like you normally would by using import.
+
+```JavaScript
```
Comment (webOS101) [inline docs/interoperability.md]: Show command line for using a different `enact create` template?  I guess we don't have a 'core' or 'ui' template.  Wonder if we should whip one up as part of public release?

File: docs/interoperability.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import { Button as BSButton} from 'reactstrap';
```
Comment (webOS101) [inline docs/interoperability.md]: Is `BSButton` the recommended way for `reactstrap`?  It seems odd to me, but I don't know `reactstrap`.

File: docs/interoperability.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+const MainPanel = kind({
+	name: 'MainPanel',
+
+	render: (props) => (
+		<Panel>
```
Comment (webOS101) [inline docs/interoperability.md]: Where did `Panel` come from?  Should we show the full imports for this?  Doesn't add anything?

File: docs/interoperability.md (hunk 2 above)
Comment (rundmt) [reply docs/interoperability.md]: Yeah this is a good idea. I think we should have something that is just a `ui` template.

File: docs/interoperability.md (hunk 3 above)
Comment (rundmt) [reply docs/interoperability.md]: Yeah I'll switch it back to button

File: docs/interoperability.md (hunk 4 above)
Comment (rundmt) [reply docs/interoperability.md]: Yeah I'll remove this and just change it to a div

**Comment by rundmt** [comment]
Yeah I think the changes you made are fine. I made some changes as well.

**Pull request merged**